Package.swift

```diff
@@ -23,13 +23,13 @@ let package = Package(
         ),
         .binaryTarget(
             name: "DittoSwift",
-            url: "https://software.ditto.live/cocoa/DittoSwift/4.5.3/dist/DittoSwift.xcframework.zip",
-            checksum: "2b7f4f8e070e38c9dc4674e98df78a76fb7e51999a1b2d6fc2f0a04d724ad567"
+            url: "https://software.ditto.live/cocoa/DittoSwift/4.5.4/dist/DittoSwift.xcframework.zip",
+            checksum: "7463a30591c9db2ca6ae37e74c4d2d2a5b19f812b29a1ce52fc54e90f5297442"
```

Comment:
The checksum I got after running `swift package compute-checksum DittoSwift.xcframework.zip` locally is different

```suggestion
            checksum: "d5ab849b194d6d4e0f66e6a3211d0cfdc94921469126484d9361b530ee5a5624"
```

Reply:
Thank you so much for catching this!